src/Screens/ItemDetail/index.tsx

```diff
@@ -1,18 +1,18 @@
 import React from "react";
-import{View,Text,TouchableOpacity,ImageBackground,SafeAreaView,Dimensions} from 'react-native'
+import{View,Text,TouchableOpacity,ImageBackground,SafeAreaView,FlatList} from 'react-native'
 import styles from "./styles";
-import  Icon from "react-native-vector-icons/FontAwesome";
-import SecondaryBtn from '../../components/UI/Button/secondaryBtn'
-
+import Icon from 'react-native-vector-icons/FontAwesome'
+import Buttonn from "../../components/button/button";
```

Comment:
Suggestion: 

Create an index file inside the root of the components directory. import all your custom components in that directory. and export them using named export. After that, you can directly access your components using syntax like below.

`import { Button } from "../../components"`

and if it's absolute import, the code will be more readable as below.

`import { Button } from "components"`